nilearn/plotting/img_plotting.py

```diff
@@ -767,8 +798,9 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
              output_file=None, display_mode='ortho', figure=None, axes=None,
              title=None, annotate=True, draw_cross=True, black_bg='auto',
              threshold=0.5, alpha=0.7, cmap=plt.cm.gist_ncar, dim='auto',
-             vmin=None, vmax=None, resampling_interpolation='nearest',
-             view_type='continuous', linewidths=2.5, **kwargs):
+             colorbar=False, cbar_tick_format="%.2g", vmin=None, vmax=None,
```

Comment:
for plot_roi don't you think "%i" could be a better default, as images contain integers?

Reply:
@NicolasGensollen just pointing out this comment in case it got buried in the discussion and we forgot about it.  I think it should be easy to address by just changing `"%.2g"` to `"%i"` here but if I'm missing something and it isn't we can leave it as it is!

Reply:
Thanks for the reminder!
I totally agree that integer formatting makes more sense as default behavior in `plot_roi`.
See 3c9afc5